src/webgpu/shader/execution/expression/call/builtin/modf.spec.ts

```diff
@@ -64,39 +89,59 @@ function makeVectorCaseWhole(kind: FPKind, v: number[]): Case {
   return { input: toVector(v, fp.scalarBuilder), expected: ws };
 }
 
+/** @returns a scalar range for a specific kind */
```

Comment:
Or maybe a object like
``` typescript
const scalar_range = {
  f32: fullF32Range,
  ...
} as const;
```

Both are fine to me

Reply:
Done